websites/F/Fire Emblem Heroes/util.ts

```diff
@@ -1,7 +1,6 @@
-export const presence = new Presence({
+export const slideshow = new Presence({
 	clientId: "1133602327476047873",
-});
-export const slideshow = presence.createSlideshow();
+}).createSlideshow();
```

Comment:
Revert these changes made to an unrelated presence. (To avoid this in the future, ensure you install the latest dependencies for this project)